hasura/metadata/cron_triggers.yaml

```diff
@@ -1,85 +1 @@
-- name: check-nominee
-  webhook: '{{HASURA_API_BASE_URL}}/cron/checkNominee'
```

Comment:
@wingfeatherwave these keep getting committed - you dont want to commit these changes, as they would disable all our cron stuff.